web-ui/styles/layout-shell.css

```
@@ -40,6 +40,7 @@ body::after {
 
 .app-shell.sidebar-collapsed {
     --side-rail-width: var(--side-rail-collapsed-width, 136px);
+    grid-template-columns: var(--side-rail-collapsed-width, 136px) minmax(0, 1fr);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)